packages/openapi-typescript/examples/github-api.ts

```diff
@@ -13321,7 +13323,7 @@ export interface components {
      * Organization ruleset conditions
      * @description Conditions for an organization ruleset
      */
-    "org-ruleset-conditions": (components["schemas"]["repository-ruleset-conditions"] & components["schemas"]["repository-ruleset-conditions-repository-name-target"]) | (components["schemas"]["repository-ruleset-conditions"] & components["schemas"]["repository-ruleset-conditions-repository-id-target"]);
+    "org-ruleset-conditions": Record<string, never> & ((components["schemas"]["repository-ruleset-conditions"] & components["schemas"]["repository-ruleset-conditions-repository-name-target"]) | (components["schemas"]["repository-ruleset-conditions"] & components["schemas"]["repository-ruleset-conditions-repository-id-target"]));
```

Comment:
Fortunately the `Record<string, never>` doesn’t interfere with the type at all; it’s just an empty object that gets ignored